package.json

```diff
@@ -62,7 +62,7 @@
     "publishedAt": "2024-02-02T14:02:52.046Z"
   },
   "engines": {
-    "node": ">=16.0.0",
+    "node": ">=18.18.0",
```

Comment:
If this is not necessary, I would lean towards reverting it.
If it is necessary, then better put it in a separate major commit.

Reply:
Done